frontend/packages/dev-console/src/components/helm/helm-utils.ts

```diff
@@ -66,6 +66,14 @@ export const filterHelmReleasesByName = (releases: HelmRelease[], filter: string
   return releases.filter((release: HelmRelease) => fuzzy(filter, release.name));
 };
 
+export const fetchHelmRelease = (
+  namespace: string,
+  helmReleaseName?: string,
```

Comment:
`helmReleaseName` should be required:
```suggestion
  helmReleaseName: string,
```